source/documentation/other-topics/troubleshooting.md

```diff
@@ -0,0 +1,111 @@
+### Troubleshooting guide
+
+#### Overview
+This document intends to give you an idea of potential troubleshooting tips and techniques to investigate and resolve application issues on the Cloud Platform. The Kubernetes project also offers a resource on [troubleshooting](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/) so we will attempt to avoid overlap. 
+
+Throughout this document we will refer to terms such as <pod_name> and <namespace>, it is assumed you'll know how to gather this information using the `kubectl get` commands.
```

Comment:
You need to escape the angle brackets, here:

<img width="799" alt="Screen Shot 2019-06-26 at 14 27 37" src="https://user-images.githubusercontent.com/2338/60183696-b6832680-981e-11e9-9d70-bf2fd809961d.png">

Reply:
:D good catch... pfft Markdown. I'll sort that out.